account/models.py

```diff
@@ -0,0 +1,15 @@
+from django.db import models
+
+class User(models.Model):
+    login_id     = models.CharField(max_length=100)
+    password     = models.CharField(max_length=100)
```

Comment:
```suggestion
    password     = models.CharField(max_length=300)
```
암호화 고려하면 300 정도로 늘려주세요.